odl/solvers/util/callback.py

```diff
@@ -315,6 +316,13 @@ def __init__(self, *args, **kwargs):
         ----------
         display_step : positive int, optional
             Number of iterations between plots. Default: 1
+        saveto : string, optional
+            Path to a directory where the figures are to be saved, potentially
+            followed by a file name. If the directory name does not exist, a
+            ``ValueError`` is raised. If not given, the figures are not saved.
+            Default: None
+        suffix : string, optinal
```

Comment:
Why is suffix a separate argument? Wouldnt getting it from the saveto string be enough?